vision/cloud-client/detect/beta_snippets.py

```diff
@@ -59,7 +59,7 @@ def localize_objects(path):
 # [END vision_localize_objects]
 
 
-# [START vision_localize_objects_uri]
+# [START vision_localize_objects_gcs]
```

Comment:
if i remember correctly the API accepts both GCS URI and HTTP URI, in this case the `gcs` suffix might not be appropriate.

Reply:
_gcs and _uri were previously inconsistently used across languages and samples. I standardized to choose _gcs when it was a GCS URI. I will keep in mind to change if it is an HTTP URI.